plugin-server/src/main/pluginsServer.ts

```diff
@@ -285,6 +285,12 @@ export async function startPluginsServer(
             )
 
             analyticsEventsIngestionConsumer = queue
+            joinAnalyticsEventsIngestionConsumer = () =>
+                queue.consumer!.join().catch((error) => {
+                    status.error('💥', 'joinAnalyticsEventsIngestionConsumer error', error)
+                    throw error
```

Comment:
or skip the helper idea and just call `closeJobs()` here instead?

Also, `closeJobs` actually does not `exit(0)` and leaves the process running in limbo, so we need another function that runs
```
        await closeJobs()
        process.exit(1)
```